app/controllers/courses_controller.rb

```diff
@@ -29,7 +29,12 @@ def index
     @self_accredited_courses = @courses_by_accrediting_provider.delete(@provider[:provider_name])
   end
 
-  def show; end
+  def show
+    @provider = Provider
+      .includes(:sites)
+      .find(params[:provider_code])
+      .first
```

Comment:
It feels to me like we should be packaging this up with the API call to get `@course` ... is the provider not already included in the response there? Can we add it?

Reply:
Good shout, let me give it a go

Reply:
Got it 👍